### ISSUE TITLE
Generate a new dataset and test the count CNN

### ISSUE THREAD

**Issue by elboyran** [NLeSC/XAI#58]
- [x] generate Dataset1Var - circles with variable radius- larger variability
- [x] Update summary notebook
- [x] Train Model1Var on Dataset1Var training subset and evaluate on the validation set
- [x] Update summary notebook
- [x] Test Model1Var on Dataset1Var test set
- [x] Update summary notebook
- [x] Generate 6 explainability heatmaps for all possible outcomes
- [x] Update summary notebook

- [x] Find out the miss-classified images and display some of them with label and prediction

- [x] Generate 6 explainability heatmaps for all possible outcomes for some miss-classified images
